### PR TITLE
Add deployment script

### PR DESCRIPTION
**Pull request by tonyyang-svail**
Fix https://github.com/sql-machine-learning/sqlflow/issues/255

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -19,15 +27,12 @@ install:
 script:
   - ls -lah
   - docker build -t sqlflow:dev -f Dockerfile.dev .
-  - docker run --rm -v $GOPATH:/go -w /go/src/github.com/sql-machine-learning/sqlflow sqlflow:dev bash -c "service mysql start && go generate ./... && go get -v -t ./... && go install ./... && go test -v ./..."
+  - docker run --rm -v $GOPATH:/go -w /go/src/github.com/sql-machine-learning/sqlflow sqlflow:dev bash scripts/test.sh
+  - bash scripts/deploy.sh
```
Comment (wangkuiyi) [inline .travis.yml]: 这里是想调用test.sh吗？

Comment (tonyyang-svail) [reply .travis.yml]: `bash scripts/deploy.sh`是应该放在deploy下的。

因为deploy只有在merge了之后才能触发，为了在PR里面测试`deploy.sh`我暂时将其放在了script下。

**Comment by tonyyang-svail** [comment]
Our deployment strategy:

1. Every merged PR will trigger a build on develop branch, therefore, update image `sqlflow/sqlflow:latest`.
1. Every nightly build will update image `sqlflow/sqlflow:nightly`.

**Review by wangkuiyi — APPROVED** [review]
Love this one!

**Pull request merged**